src/components/SingleBubble/SingleBubble.js

```diff
@@ -3,21 +3,22 @@ import PropTypes from 'prop-types';
 import './SingleBubble.scss';
 
 class SingleBubble extends Component {
+  // NOTE: Changed the specification, the component doesn't really requires the y co-ordinate
```

Comment:
Cool, comment can be removed

Reply:
see: https://github.com/digirati-co-uk/timeliner/pull/56/commits/be1f0005f9fec3e46f3c412d813075eb111a7edc